client/state/ui/olark/actions.js

```diff
@@ -7,7 +7,8 @@ import {
 	OLARK_REQUEST,
 	OLARK_TIMEOUT,
 	OLARK_OPERATORS_AVAILABLE,
-	OLARK_OPERATORS_AWAY
+	OLARK_OPERATORS_AWAY,
+	OLARK_SET_AVAILABILITY,
```

Comment:
The eternal question :)
I kinda wish we had an eslint rule for it one way or the other, but as far as I can tell it's still a matter of preference in Calypso. I'm in the yes to dangling comma camp, only because it makes for more readable diffs.